data/skin_reaction/meta.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name: skin_reaction
+description: |-
+    Repetitive exposure to a chemical agent can induce an immune reaction
+    in inherently susceptible individuals that leads to skin sensitization. The
+    dataset used in this study was retrieved from the ICCVAM (Interagency Coordinating
+    Committee on the Validation of Alternative Methods) report on the rLLNA.
+targets:
+    - id: Skin_Reaction
+      description: whether it can cause skin reaction (1) or not (0).
+      units: Skin_Reaction
+      type: categorical
+      names:
+          - Skin Reaction
+          - skin sensitization
+          - agent Induced Skin Reaction
+          - drug induced skin immune reaction
+      uris:
+          - http://purl.bioontology.org/ontology/SNOMEDCT/264519003
```

Comment:
```suggestion
```

Reply:
I'd only keep maps to definitions that are really matching 99%

Reply:
removed